src/main/webapp/index.xhtml

```diff
@@ -34,6 +34,19 @@
       xmlns:pt="http://xmlns.jcp.org/jsf/passthrough"
 >
 
+<f:event type="preRenderView"
```

Comment:
For setting the response headers, this is more readable and more performant:

```
@WebFilter(urlPatterns = "*.xhtml")
public class HeaderFilter implements Filter {

    @Override
    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
        HttpServletResponse servletResponse = (HttpServletResponse) response;
        servletResponse.setHeader("X-XSS-Protection", "1; mode=block");
        ....
        chain.doFilter(request, response);
    }
...
}
```

Reply:
@rdebusscher Thank! Will do.